.gitignore

```
@@ -40,3 +40,5 @@ junit.xml
 .envrc.local
 
 .idea/
+
+copy.package.json
```

nit: it's usually better to add a suffix to the file name so the files end up next to each other in local dev
```suggestion
package.copy.json
```
or so.